controllers/profile/profile.go

```diff
@@ -88,10 +108,60 @@ func (r *Reconciler) Reconcile(req reconcile.Request) (reconcile.Result, error)
 		return reconcile.Result{}, errors.Wrap(ignoreNotFound(err), errGetProfile)
 	}
 
+	for name, contents := range profile.Data {
+		profilePath, err := getProfilePath(name, profile)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+
+		saveProfileOnDisk(profilePath, contents)
```

Comment:
I think we should handle the returned error.

Reply:
good spot, I completely overlooked this before - it should now be handled now.